src/pages/Admin/ToolsPage/scripts/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./analytics/config";
+export * from "./makeWorldOwner/config";
```

Comment:
Shouldn't these be `./analytics` and `./makeWorldOwner`?
Is adding `index.ts` in them going to mess up anything?

Reply:
Done